Team page completed

pAditya198 opened the pull request:
![Screenshot (72)](https://user-images.githubusercontent.com/42460356/62892488-09308780-bd66-11e9-9f4d-5ecab03bb897.png)

parth181195 commented:
@pAditya198 can you make different component for profile page icon cause it will not open pop up the model

parth181195 commented:
we also need to make some changes to input config
like we need to pass team member details in config, it only needs to have name and role plus profile links

pAditya198 commented:
@parth181195 the profile page icon component that you are talking about is it similar to the speakerconfig or the speaker dialogue.

pAditya198 commented:
![Screenshot (73)](https://user-images.githubusercontent.com/42460356/62972019-be2f7680-be30-11e9-99eb-d8d1ea06801b.png)

@rossoskull @aashutoshrathi @parth181195  please review 
I've modified the Font and created a config for team as well as modified the team-card in the elements section for seperate data, and removed the pop-up in the team page.

aashutoshrathi commented:
Can we atleast ensure this is Open Sans?

pAditya198 commented:
@aashutoshrathi yes I've ensured it online

aashutoshrathi commented on src/app/components/team-page/team-page.component.html:
```diff
@@ -1 +1,47 @@
+      <div class="col-xs-12">
+        <app-homepage-title title='Team' color='#419BF9'></app-homepage-title>
+      </div>
+      <p>DevFest Gandhinagar is brought to you with 💓 love,
```
Can you replace this emoji with the simple :heart: emoji

pAditya198 replied:
I tried but couldn't because instead of red heart it displays white heart.

pAditya198 replied:
@aashutoshrathi modified as per required please review again.

pAditya198 commented:
![Screenshot (75)](https://user-images.githubusercontent.com/42460356/63009861-f66f9d00-bea2-11e9-9f1b-72e7b2854812.png)

 Name and Post font weight changed and also added the red heart emoji
